osrf/vrx-docker
# 2023 minimum working example

Providing files that match/ support the new competitor example image.  This could be ready for review as-is, but I'm marking it draft since I'm not sure we want to merge it before the rest of the docker work is complete and testable, or whether we want to move the image elsewhere.
I ran vrx from my system directly as I can't get dockwater working at the moment.  I also did not test with vrx-server as I'm not sure what its status is.

To test;
* grab the image
* `docker run -it jessicaherman/vrx-competitor-example:2023.v2` - you should see a note that says the node is starting
* In another terminal, run `ros2 launch vrx_gz competition.launch.py world:=stationkeeping_task`
* Once the task message starts to publish you should see information in the competitor window.  When the status changes to "running", you should see that the vrx terminal reports the bridge having translated a message.  The wamv will move forward slightly, then the thrusters will stop.
* Ctrl-C out of the vrx window before the task terminates.
* Run `ros2 launch vrx_gz competition.launch.py world:=perception_task`
* Let the task run to completion and note that the score is no longer at 10.
